js/recruitment.js

```diff
@@ -36,18 +36,15 @@ function onRecruitmentLoaded(data) {
 
   let availableSeatsInBand = {
     "Kjellerbandet": 
-      "KB har opptak på trompet! Ta kontakt hvis du er interessert.",
-    "Leisure Suit Lovers": 
-      "Er du interessert i Disco? Liker du ABBA, Earth, Wind & Fire og andre 70/80-talls hits? Søk Leisure Suit Lovers! LSL søker discoglade musikere på keys og gitar, og en korist til H21! Søk søk søk!",
-    "S. Møller Storband":
-      "S. Møller har opptak på tenorsaks, altsaksofon, trompet og trombone, og ønsker at interesserte sender mail med opptak av seg selv + litt info om hvem de er og hva de holder på med. Aktuelle søkere kan bli innkalt til fysisk prøvespill, men det vurderer vi etter antall søkere.",
+      "KB har opptak på piano! Ta kontakt hvis du er interessert.",
+    "Leisure Suit Lovers": null, // "Er du interessert i Disco? Liker du ABBA, Earth, Wind & Fire og andre 70/80-talls hits? Søk Leisure Suit Lovers! LSL søker discoglade musikere på keys og gitar, og en korist til H21! Søk søk søk!",
+    "S. Møller Storband": null, // "S. Møller har opptak på tenorsaks, altsaksofon, trompet og trombone, og ønsker at interesserte sender mail med opptak av seg selv + litt info om hvem de er og hva de holder på med. Aktuelle søkere kan bli innkalt til fysisk prøvespill, men det vurderer vi etter antall søkere.",
```

Comment:
Vent, fjern disse kommentarene. De har jo null funksjon, vi har jo git for å huske hva som sto der før.